patches/coveo.analytics+2.30.38.patch

```diff
@@ -1,17 +1,8 @@
 diff --git a/node_modules/coveo.analytics/dist/browser.mjs b/node_modules/coveo.analytics/dist/browser.mjs
-index 3de0ed7..b7d2c0e 100644
+index 78efccc..7702f2d 100644
 --- a/node_modules/coveo.analytics/dist/browser.mjs
 +++ b/node_modules/coveo.analytics/dist/browser.mjs
-@@ -1093,8 +1093,6 @@ class NoopAnalyticsClient {
-     }
- }
- 
--const fetch$1 = window.fetch;
```

Comment:
We need this: Otherwise it will break on NodeJS

Reply:
It indeed does whenever the code is executed outside the browser. :+1: